aten/src/ATen/native/MathBitsFallback.h

```diff
@@ -98,7 +98,9 @@ struct MathOpFallback {
           continue;
         }
         auto tensor = std::move(ivalue).toTensor();
-        TORCH_CHECK_NOT_IMPLEMENTED(!tensor.is_meta(), op_name, " fallback does not support meta tensors.");
+        if (tensor.is_meta()) {
```

Comment:
@ezyang can we enable `TestMathBits` for meta?

Reply:
Sure I don't see why not. But I am suspicious about continue'ing here; it seems like we should run the physical conjugation for compilers tracing through this.

Reply:
oh yeah sorry it should be below the clone. just wanted to skip the std::move below

Reply:
The continue still doesn't look right. At the very least the fallback would not support multiple mutable tensors with conj bit set. But it seems to me that we are obligated to run all of the logic below. Some tests would help.